spec/support/shared/server_selector.rb

```diff
@@ -231,3 +231,12 @@
     end
   end
 end
+
+shared_examples 'a server selector with legacy secondary_ok' do
+  describe '#slave_ok?' do
+    it 'behaves like #secondary_ok?' do
+      selector = Mongo::ServerSelector::Nearest.new
+      expect(selector.slave_ok?).to eq(selector.secondary_ok?)
```

Comment:
I am confused by this test. If this simply asserts that slave_ok is the same as secondary_ok then why is selector specified to be nearest? Shouldn't this apply to all selector types?